code/__DEFINES/rust_g.dm

```diff
@@ -320,5 +320,3 @@
  */
 #define rustg_worley_generate(region_size, threshold, node_per_region_chance, size, node_min, node_max) \
 	RUSTG_CALL(RUST_G, "worley_generate")(region_size, threshold, node_per_region_chance, size, node_min, node_max)
-
-
```

Comment:
This whitespace diff is offending me, but it will get thrown back next time I update RUSTG, which will probably be soon.

Reply:
It's there to taunt you